recipes/ruby/all/conanfile.py

```diff
@@ -1,22 +1,18 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os, to_apple_arch
+from conan.tools.build import cross_building
+from conan.tools.files import apply_conandata_patches, collect_libs, copy, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsDeps, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime, msvc_runtime_flag, unix_path, VCVars
+from conan.tools.scm import Version
+
 import glob
 import os
 import re
 
-from conan import ConanFile
-from conan.tools.apple.apple import is_apple_os, to_apple_arch
-
-try:
-    from conan.tools.cross_building import cross_building
-except ImportError:
-    from conan.tools.build.cross_building import cross_building
-
-from conan.tools.files import apply_conandata_patches
-from conan.tools.gnu import Autotools, AutotoolsDeps, AutotoolsToolchain
-from conan.tools.microsoft import msvc_runtime_flag, is_msvc
-from conans import tools
-from conans.errors import ConanInvalidConfiguration
-
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.51.3"
```

Comment:
```suggestion
required_conan_version = ">=1.53"
```
For `self.settings.rm_safe()` support.